data_set/advanced/add.md

```diff
@@ -48,6 +52,10 @@ Create a new **data set profile**.
 #### ❌ Raises
 * `SecurityRequestError`<br>
   Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`.
+ * `AddOperationError`<br>
+  Raises `AddOperationError` when the **data set profile** cannot be added because it already exists in the environment.
```

Comment:
```markdown
 * `AddOperationError`<br>
  Raises `AddOperationError` when the **data set profile** cannot be added because it already exists.
```

Make the same updates everywhere else this exception is documented.

Reply:
Agreed. Changed.